### DISCUSSION TITLE
Octernship eligibility

### DISCUSSION THREAD

**Discussion by 20481A05K0** [community/community#63394] [GitHub Education]
### Hi everyone,

Even though i am student it shows you need to contribute for open source projects

**Comment by Shr11** [comment]
Hey there! yes it does need you to be an active contributor but that doesn't mean you have to code for it, you can also do no-code contributions like documentation, answering queries or providing translation in your native language etc.

Hope this helps!

**Reply by nirban256** [reply]
do I need to contribute in specific repos or in general I have to contribute?

**Reply by Shr11** [reply]
You should contribute to repos with Hacktoberfest tag in their topics for them to be counted, which you can easily search in the search by topic space in github.

**Reply by garvcodes** [reply]
Can I do it if I live in the US??

**Comment by Suraj-kumar00** [comment]
> do I need to contribute in specific repos or in general I have to contribute?

Hi @nirban256, Can you please elaborate your question so that I can answer!

**Reply by nirban256** [reply]
Yes, @Suraj-kumar00  I am asking that to be eligible for the github octernship, are there any specific set of repositories, fields in which I have to contribute to become eligible. If so then what are the fields and repositories, can you just tell me?

**Reply by Shr11** [reply]
As far as I know, no there aren't any such repositories. It just checks how active you've been for the past six months or so in github contributions at any level.

**Comment by shivang21007** [comment]
> ### Hi everyone,
> Even though i am student it shows you need to contribute for open source projects
> 
> ### A suggestion...(optional)
> _No response_

Yes, I have also a same question, It is written there that you should be a active contributor?
Does it means you should maintain a certain streak or something?

**Reply by Shr11** [reply]
I believe Streak has to be maintained yes, as I was eligible for some octernship projects last year , however this year since  my recent record wasn't much , I was disqualified.

**Reply by shivang21007** [reply]
Thanks @Shr11 for clarification.
